Procfile

```diff
@@ -1 +1 @@
-release: npm i -g fastly-cli && fastly purge-all -k $FASTLY_PURGE_KEY -s $FASTLY_SERVICE_ID
+release: curl -XPOST -H "Fastly-Key:$FASTLY_PURGE_KEY" https://api.fastly.com/service/$FASTLY_SERVICE_ID/purge_all
```

Comment:
so the reason we need to change it is because whatever context this is being run, npm/node is not actually available to us at this point. If we leave this in it says "npm not found"

I know it's odd because node is clearly available to us in the compile step 😂